test/mongodb.test.js

```diff
@@ -1611,6 +1612,67 @@ describe('mongodb connector', function() {
       });
     });
 
+    it('find should be able to query around location in deep/multiple keys', function(done) {
+      var coords = { lat: 1.25, lng: 20.20 };
+
+      geoDb.autoupdate(function(err) {
+        var postNb = 0;
+
+        var createPostWithLocation = function(callback) {
+          postNb++;
+
+          PostWithLocation.create({
+            title: 'Post #' + postNb,
+            location: new GeoPoint({
+              lat: coords.lat + (Math.random() * 2 - 1),
+              lng: coords.lng + (Math.random() * 2 - 1),
+            }),
+          }, callback);
+        };
+
+        async.parallel([
+          createPostWithLocation,
+          createPostWithLocation,
+          createPostWithLocation,
+          createPostWithLocation,
```

Comment:
Why are there exactly four posts created? It's a good practice to keep the test data as minimal as possible. Is it enough to have two model instances for this test?